### PR TITLE
fix(data-warehouse): Fixed react hook ordering

### PR DESCRIPTION
**Pull request by Gilbert09**
## Problem
- "Rendered fewer hooks than expected. This may be caused by an accidental early return statement."
- [Sentry error](https://posthog.sentry.io/issues/5481492964/?referrer=slack&notification_uuid=b2b37ef2-63ed-44cf-baab-22c03135d9c3&alert_rule_id=1075039&alert_type=issue)

## Changes
Fixed it m8

## Does this work well for both Cloud and self-hosted?
yup

## How did you test this code?
Locally

### CODE REVIEW

File: frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx [hunk 1]
```diff
@@ -174,19 +174,23 @@ function DefinitionView({ group }: { group: TaxonomicFilterGroup }): JSX.Element
             </>
         )
     }
-    if (isProperty) {
+
+    const hasSentAsLabel = useMemo(() => {
```
Comment (neilkakkar) [inline frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx:178]: I think this still will bork because of the early returns above?

Comment (Gilbert09) [reply frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx:178]: Oh poop, I didn't see those, one sec..

Comment (Gilbert09) [reply frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx]: @neilkakkar sorted

**Review by neilkakkar — APPROVED** [review]
Noice makes sense!

**Pull request merged**